Apps/Sandcastle/gallery/3D Models.html

```diff
@@ -28,7 +28,6 @@
     'use strict';
 //Sandcastle_Begin
 var viewer = new Cesium.Viewer('cesiumContainer', {
-    scene3DOnly: true,
```

Comment:
Can you check all the Sandcastle examples with `scene3DOnly: true`.  Perhaps the development on and CZML examples?